src/main/java/Frank.java

```diff
@@ -0,0 +1,71 @@
+import java.util.Scanner;
+public class Frank {
+    public static void main(String[] args) {
+
+        String solidLine = "\n------------------------------------------------------------------------------------------------------------------------------\n";
```

Comment:
May want to make this final as this string will not be changed. Similar issue in other string literal
```suggestion
        final String solidLine = "\n------------------------------------------------------------------------------------------------------------------------------\n";
```